tensorbay/cli/auth.py

```diff
@@ -132,17 +136,48 @@ def _update_profile(config_parser: ConfigParser, profile_name: str, arg1: str, a
     click.echo("".join(messages))
 
 
-def _check_args_and_options(arg1: str, arg2: str, get: bool, unset: bool, is_all: bool) -> None:
-    if is_all and not (get or unset):
-        error('Use "--all" option with "--get" or "--unset" option')
+def _check_args_and_options(  # pylint: disable=too-many-arguments
+    arg1: str, arg2: str, get: bool, status: bool, unset: bool, is_all: bool
+) -> None:
+    if is_all and not (get or unset or status):
+        error('Use "--all" option with "--get", "--unset" or "--status" option')
 
-    if get and unset:
-        error('Use either "--get" or "--unset"')
+    if get and (unset or status) or (unset and status):
+        error('Use only one of "--get", "--unset" and "--status"')
 
-    if (get or unset) and (arg1 or arg2):
+    if (get or unset or status) and (arg1 or arg2):
         error("Option requires 0 arguments")
 
 
 def _echo_formatted_profile(name: str, value: str) -> None:
     formatted_value = indent(value, INDENT)
     click.echo(f"{name} = {formatted_value}\n")
+
+
+def _status_auth(obj: Dict[str, str], config_parser: ConfigParser, is_all: bool) -> None:
+    if is_all:
+        try:
+            for key, value in config_parser["profiles"].items():
+                _echo_user_info(key, value)
+        except KeyError:
+            pass
+        return
+
+    profile_name = obj["profile_name"]
+    try:
+        _echo_user_info(profile_name, config_parser["profiles"][profile_name])
+    except KeyError:
+        error(f"Profile '{profile_name}' doesn't exist.")
+
+
+def _echo_user_info(key: str, value: str) -> None:
+    auth_info = value.split()
+    access_key = auth_info[0]
+    url = auth_info[1] if len(auth_info) > 1 else ""
+    user_info = get_gas(access_key, url, key).get_user()
+    information = [f"USER: {user_info.name}"]
+    team = user_info.team
+    if team:
+        information.append(f"TEAM: {team.name}")
```

Comment:
can we `click.echo` the message directly rather than join the message before `echo`?